toolkit/resources/assets/isomaker/iso_root_static_files/boot/grub2/grub.cfg

```diff
@@ -2,7 +2,7 @@ set timeout=0
 
 # The use of mariner.media=CDROM is a workaround that our installer does not require
 # but it is observed to be needed to boot on some hardware/SoCs.
-menuentry "CBL-Mariner" {
+menuentry "Azure Linux" {
```

Comment:
Ask @Chris Gunn or @Gary Swalling what is best here.

Reply:
This change looks good - it changes grub's boot menu entry string that is displayed for this entry

Reply:
Thanks for verifying